sdk/eventhub/event-hubs/test/receiver.spec.ts

```diff
@@ -99,27 +94,24 @@ describe("EventHub Receiver", function(): void {
           stamp: uid
         }
       };
+
+      const offset = (await client.getPartitionInformation(partitionId)).lastEnqueuedOffset;
       await client.send(ed, partitionId);
       debug(">>>>>>> Sent the new message after creating the receiver. We should only receive this message.");
-      const data2 = await breceiver.receive(10, 20);
+      const data2 = await client.receiveBatch(partitionId, 10, 20, {
```

Comment:
data2 -> data
Same in a few other places

Reply:
Got it, I was wondering if we should do one round of sole clean / improve, refactor of all eventhub tests PR and keep changes to it? That way we can get past the functional issues for now